### PR TITLE
Move YACL dependancy check to config screen

### PR DESCRIPTION
**Pull request by Bawnorton**
TL;DR: No one reads the logs
Edit: See #86 for merged commit

### CODE REVIEW

**Review by Benonardo — APPROVED** [review]

**Comment by enjarai** [comment]
Next release will come from https://github.com/enjarai/do-a-barrel-roll/tree/1.19.4/next, so I'll move this PR there.

File: src/main/java/nl/enjarai/doabarrelroll/compat/yacl/YACLImplementation.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                                .option(getOption(Double.class, "banking", "banking_strength", false)
+                                        .controller(option -> new DoubleSliderController(option, 0.0, 100.0, 1.0))
+                                        .binding(20.0, () -> ModConfig.INSTANCE.getBankingStrength(), value -> ModConfig.INSTANCE.setBankingStrength(value))
+                                        .build())
+                                .build())
+                        .group(OptionGroup.createBuilder()
+                                .name(getText("thrust"))
+                                .collapsed(true)
+                                .option(getBooleanOption("thrust", "enable_thrust", true)
+                                        .binding(false, () -> ModConfig.INSTANCE.getEnableThrust(), value -> ModConfig.INSTANCE.setEnableThrust(value))
```
Comment (enjarai) [inline src/main/java/nl/enjarai/doabarrelroll/compat/yacl/YACLImplementation.java:64]: You shouldn't be using the getters and setters for the config itself. They take other things into account, like if the server allows it or not. In this case, that could fuck with the config.

**Review by enjarai — CHANGES_REQUESTED** [review]
Actually, this wont merge with https://github.com/enjarai/do-a-barrel-roll/tree/1.19.4/next easily at all. Could you redo this while basing it off that branch? Also check the comment I left on `YACLImplementation`. 

If its too much trouble, I'll probably fix this myself at some point in the future.

**Comment by Bawnorton** [comment]
See #86

**Pull request closed without merging**